packages/cfd/src/Constants/routes-config.tsx

```diff
@@ -1,32 +1,32 @@
 import React from 'react';
```

Comment:
Is it a React component or just a constant file?
should it be `routes-config.ts`

Reply:
This file has a jsx expression in it that's why we have a 'tsx' extension.